sklearn/feature_extraction/text.py

```diff
@@ -371,10 +371,12 @@ class HashingVectorizer(BaseEstimator, VectorizerMixin, TransformerMixin):
         values are 'ignore' and 'replace'.
 
     strip_accents : {'ascii', 'unicode', None}
-        Remove accents during the preprocessing step.
+        Remove accents and perform other character normalization
+        during the preprocessing step.
         'ascii' is a fast method that only works on characters that have
         an direct ASCII mapping.
         'unicode' is a slightly slower method that works on any characters.
+        It uses NFKD normalization from :func:`unicodedata.normalize`.
```

Comment:
Seems that acsii (strip_accents_ascii) also uses unicodedata.normalize?
Maybe current doc is enough? (Seems that the main difference between ascii and unicode is ascii only works on characters that have an direct ASCII mapping.)

Reply:
I have seen people run a unicode normalization before feeding the data to TfidfVectorizer not realizing there was an option for it. So I think it's good to be explicit about it in the docs.